crates/libs/core/src/guid.rs

```diff
@@ -21,7 +21,14 @@ pub struct GUID {
 impl GUID {
     /// Creates a unique `GUID` value.
     pub fn new() -> Result<Self> {
-        unsafe { imp::CoCreateGuid() }
+        let mut guid = Self::zeroed();
+        let result = unsafe { imp::UuidCreate(&mut guid as *mut _ as _) };
+
+        if result == 0 {
+            Ok(guid)
+        } else {
+            Err(Error::from_hresult(HRESULT::from_win32(result as u32)))
+        }
```

Comment:
`UuidCreate` returns multiple success values (e.g., `RPC_S_UUID_LOCAL_ONLY`). The `CoCreateGuid` wrapper swallows this one specifically and returns `S_OK`.

Reply:
On my copy of 25H2 26200.5670, I verified it has a check for `RPC_S_UUID_LOCAL_ONLY || RPC_S_OK` and returns `S_OK`.

See also: `onecore\com\combase\class\cocrguid.cxx`

Reply:
OK, I've preserved that behavior.